readme.md

```diff
@@ -107,7 +116,18 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
-
+app.get('/', function(req, res){
+  res.send('show');
+});
+app.post('/', function(req, res){
+  res.send('create');
+});
+app.put('/', function(req, res){
+  res.send('update');
+});
+app.delete('/', function(req, res){
+  res.send('delete');
+});
```

Comment:
Close.  Which item would we be showing, updating, or deleting?  We need more info in the url.